core/assets/bundles/bundle_fr.properties

```
@@ -1650,11 +1650,11 @@ block.large-constructor.description = Fabrique des structures d'une taille maxim
 block.deconstructor.name = Déconstructeur
 block.deconstructor.description = Déconstruit les structures et les unités. Retourne 100% du coût de construction.
 block.payload-loader.name = Chargeur de charge utile
-block.payload-loader.description = Chargez les liquides et les articles dans les blocs.
+block.payload-loader.description = Charge les liquides et les ressources dans les blocs.
 block.payload-unloader.name = Déchargeur de charge utile
-block.payload-unloader.description = Décharge les liquides et les articles des blocs.
+block.payload-unloader.description = Décharge les liquides et les ressources des blocs.
 block.heat-source.name = Source de Chaleur
-block.heat-source.description = Produit de grandes quantités de chaleur. Bac à sable uniquement.
+block.heat-source.description = Produit de grandes quantités de chaleur. (Bac à sable uniquement)
```

```suggestion
block.heat-source.description = Produit de grandes quantités de chaleur. Bac à sable uniquement.
```
Non, aucun autre traduction ne le fait et c pas dans le bundle original.
https://github.com/phinner/mindustry/blob/e4253bcbf8fdc638cc52247ac553b5ed697447d4/core/assets/bundles/bundle.properties#L1657